qa/workunits/cephtool/test.sh

```diff
@@ -155,6 +155,37 @@ function expect_config_value()
   fi
 }
 
+function ceph_watch_start()
+{
+    local whatch_opt=--watch
+
+    if [ -n "$1" ]; then
+	whatch_opt=--watch-$1
+    fi
+
+    CEPH_WATCH_FILE=${TMPDIR}/CEPH_WATCH_$$
+    ceph $whatch_opt > $CEPH_WATCH_FILE &
+    CEPH_WATCH_PID=$!
+}
+
+function ceph_watch_wait()
+{
+    local regexp=$1
+    local timeout=30
+
+    if [ -n "$2" ]; then
+	timeout=$2
+    fi
+
+    for i in `seq ${timeout}`; do
+	sleep 1
+	grep -q "$regexp" $CEPH_WATCH_FILE && break
+    done
+
+    kill $CEPH_WATCH_PID
+    grep "$regexp" $CEPH_WATCH_FILE
```

Comment:
The file is created in $TMPDIR, which is always removed on exit. I'd rather not add this additional remove as it might give wrong impression that the cleanup is supposed to be here, while it may fail, e.g. when the test is abnormally interrupted.

Note,  there is no this additional remove in the current version too.